bin/nib-ls

```diff
@@ -22,6 +22,7 @@ import sys
 from math import ceil
 from optparse import OptionParser, Option
 from io import StringIO
+from collections import Counter
```

Comment:
Yeap... Would be much slower happen I feed some float volume in by mistake ;-)

On April 6, 2016 6:25:11 PM EDT, Chris Markiewicz <notifications@github.com> wrote:
>> @@ -22,6 +22,7 @@ import sys
>>  from math import ceil
>>  from optparse import OptionParser, Option
>>  from io import StringIO
>> +from collections import Counter
>
>Below you have:
>
>```Python
>counter = Counter()
>counter.update(d.flatten())
># go through each entry and report only non-0 ones
>row += ["@l" + " ".join("%g:%d" % (i, counter[i]) for i in
>sorted(counter))]
>```
>
>You could do something like:
>
>```Python
>row += ["@l" + " ".join("%g:%d" % (i, np.sum(d == i)) for i in
>np.unique(d))]
>```
>
>I suspect this would be slower than Counter, so take it for whatever
>it's worth.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nipy/nibabel/pull/437/files/0851f93c446db78ce7515bb5966ffa91dd043ba4#r58792848